doc/nrf/index.rst

```diff
@@ -30,6 +30,7 @@ A "99" at the end of the version number of this documentation indicates continuo
    scripts
    integrations
    releases_and_maturity
+   vulnerabilities
```

Comment:
Kindly please place this page under the Security section ` doc/nrf/security`, with index updated in  `doc/nrf/security.rst`

Reply:
We discussed this a bit more with Gaute and Uma and we think a better place for it would be under Releases and Maturity. Is this ok with you @greg-fer ?

Reply:
Up to you, as long as it is not on the main level :)

Reply:
The page is now located under release_and_maturity.